### PR TITLE
Import catalog

### PR DESCRIPTION
**Pull request by aditya-kapoor**
merge after #1 

# Specs remaining, but the basic idea is ready for review.

### CODE REVIEW

File: lib/RDC/item.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module RDC
+  class Item
+    attr_accessor :sku, :legacy_sku, :value, :price, :restriction, :restriction_detail
+
+    def initialize(hash_values)
+      puts "Hash Values :: #{hash_values}"
```
Comment (davidrichey) [inline lib/RDC/item.rb:8]: Probably remove this `puts`

File: lib/RDC/restaurant.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+require_relative 'item'
+
+module RDC
+  class Restaurant
+    attr_accessor :name, :items, :short_description, :long_description,
+                  :online, :address, :image
+
+    def initialize(hash_values)
+      puts "Initializing Restaurant #{hash_values['RestaurantName']}"
```
Comment (davidrichey) [inline lib/RDC/restaurant.rb:11]: And this one

**Review by davidrichey — APPROVED** [review]
This looks good just had some extra puts statements we probably don't want

**Comment by aditya-kapoor** [comment]
@davidrichey I will remove this once the exported CSV looks good to everyone.

wdyt?

**Comment by aditya-kapoor** [comment]
Probably when we move to ordering on RDC to ascertain the CSV is indeed correct.

**Pull request merged**